src/amplitude_experiment/local/client.py

```diff
@@ -85,10 +113,11 @@ def __setup_connection_pool(self):
 
     def close(self) -> None:
         """
-        Close resource like connection pool with client
+        Stop polling for flag configurations. Close resource like connection pool with client
         """
         self.poller.stop()
         self._connection_pool.close()
+        self.is_running = False
```

Comment:
why do we need is_runing at the client level?

Reply:
That's to support we can call `start` multiple times. Right now the start each time will trigger a fetch, I guess still optional though, got reminded from our earlier discussion

Reply:
Another extra thought and would take this down, multiple start will just be multiple fetch and seems not hurting poller.